docs/source/using/mounting-host-folders.adoc

```diff
@@ -0,0 +1,122 @@
+[[mounting-host-folders]]
+= Mounting host folders
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[mount-host-folders-overview]]
+== Overview
+
+Some drivers mount a host folder within the VM so that you can share files between the VM and the host.
+These folders are not currently configurable, and are different for each driver and OS that you use.
+
+[cols=",,,",options="header",]
+|========================================
+|Driver |OS |HostFolder |VM
+|Virtualbox |Linux |/home |/hosthome
+|Virtualbox |OSX |/Users |/Users
+|Virtualbox |Windows |C://Users |/c/Users
+|VMWare Fusion |OSX |/Users |/Users
+|Xhyve |OSX |/Users |/Users
+|========================================
+
+NOTE: Host folder sharing is not implemented in the KVM and Hyper-V drivers. You can
+link:#mounting-custom-shared-folders[mount a CIFS-based shared folder] inside the VM instead.
+
+[[mounting-custom-shared-folders]]
+== Mounting custom shared folders
```

Comment:
Looking at the section as a standalone file, I don't think we need this title. The section below it can be a part of the overview, and actually I would probably raise the CIFS and SSHFS headings up one level so that they can appear in the navigation.